src/cpp/src/utils.cpp

```diff
@@ -97,6 +96,25 @@ void update_npu_config(ov::AnyMap& config,
     rename_key(config, "++SHARED_HEAD_CONFIG", "++NPUW_LLM_SHARED_HEAD_CONFIG");
 }
 
+void update_npu_config_whisper(ov::AnyMap& config,
+                               const ov::genai::utils::KVAxesPosition& kv_pos,
+                               const ov::genai::utils::KVDesc& kv_desc) {
+    update_config(config, {"NPU_USE_NPUW", "YES"});
+    update_config(config, {"NPUW_ONLINE_PIPELINE", "NONE"});
+    update_config(config, {"NPUW_FUNCALL_FOR_ALL", "NO"});
+    update_config(config, {"NPUW_FOLD", "NO"});
```

Comment:
To enable weight sharing for FP16 models:
```suggestion
    update_config(config, {"NPUW_FUNCALL_FOR_ALL", "YES"});
    update_config(config, {"NPUW_FOLD", "YES"});
    update_config(config, {"NPUW_WEIGHTS_BANK", "whisper-shared"});
```
On top of that, for INT8-SYM:
```
    update_config(config, {"NPUW_DQ", "YES"});
    update_config(config, {"NPU_COMILER_DYNAMIC_QUANTIZATION", "YES"});
```

For asym, we need to consider a third option (CWAI?)